### PR TITLE
added info to the maintain-website-guide.md file about how to use the old website scraped data templates

### PR DESCRIPTION
**Pull request by skaser85**
This is to add instructions to the maintain-website-guide.md about how to download a template directory from the data scraped from the old website.  Please let me know if you'd like anything changed.

### CODE REVIEW

File: content/pages/guides/maintain-website-guide.md [hunk 1]
```diff
@@ -8,6 +8,52 @@ You want to help with integrating new content into the site? Great to hear that!
 
 A new Coding Train video has been released and you want to add it to the website so everyone else can find it there? OK, just follow these steps:
 
+## Coding Challenges - starting from a template
+
+Data from the old website was scraped into starter templates that reside in a GitHub repo that can be found [here](https://github.com/skaser85/thecodingtrain_website_data).  Each template directory includes three items:
```
Comment (shiffman) [inline content/pages/guides/maintain-website-guide.md:13]:
```suggestion
Data from the old website was scraped into starter templates that [reside in a GitHub repo that can be found here](https://github.com/skaser85/thecodingtrain_website_data).  Each template directory includes three items:
```

**Review by shiffman — COMMENTED** [review]
This is fantastic! I'd love to add a short section about what generally needs to be updated and added, for example:

* Update links to code examples (p5 web editor for p5, new Coding Challenges repo for Processing)
* Add thumbnail images for code examples
* Remove broken showcase links
* Add thumbnail images for showcase
* Add additional metadata descriptions for related links and videos
* Replace any links to old website with new ones.

I can merge this now and we can track this later in a separate issue or let me know if you would like to integrate before merging! 

Thank you for this heroic effort!!

File: content/pages/guides/maintain-website-guide.md (hunk 1 above)
Comment (shiffman) [reply content/pages/guides/maintain-website-guide.md:13]: This is just a small note that having more description text in a link is helpful for accessibility / screen readers.

Comment (skaser85) [reply content/pages/guides/maintain-website-guide.md]: Great point!  I'll keep that in mind in the future.

**Comment by skaser85** [comment]
> This is fantastic! I'd love to add a short section about what generally needs to be updated and added, for example:
> 
> * Update links to code examples (p5 web editor for p5, new Coding Challenges repo for Processing)
> * Add thumbnail images for code examples
> * Remove broken showcase links
> * Add thumbnail images for showcase
> * Add additional metadata descriptions for related links and videos
> * Replace any links to old website with new ones.
> 
> I can merge this now and we can track this later in a separate issue or let me know if you would like to integrate before merging!
> 
> Thank you for this heroic effort!!

Thanks, Dan.  

I'd say we can track this in a new issue.  I should have time tomorrow afternoon after work to add those requested items.

**Comment by shiffman** [comment]
Great, thank you!

**Pull request merged**